dataframe_api_compat/pandas_standard/__init__.py

```diff
@@ -104,12 +104,15 @@ def map_pandas_dtype_to_standard_dtype(dtype: Any) -> DType:
         return Namespace.Float32()
     if dtype == "Float32":
         return Namespace.Float32()
-    if dtype == "bool":
+    if dtype in ("bool", "boolean"):
+        # Also for `pandas.core.arrays.boolean.BooleanDtype`
         return Namespace.Bool()
     if dtype == "object":
         return Namespace.String()
     if dtype == "string":
         return Namespace.String()
+    if not hasattr(dtype, "startswith"):
+        dtype = str(dtype)
```

Comment:
is it possible to do this in a less hacky way?

Reply:
We can try to use `name` attribute if it exists.